bin/embedding-sdk/generate-sdk-package-files.js

```diff
@@ -67,14 +67,18 @@ function generateSdkPackage() {
   );
 }
 
-function generateLicenseFile() {
+/**
+ * @param {string} source
+ * @param {string} target
+ */
+function copyFileToOutput(source, target = source) {
   const licenseContent = fs.readFileSync(
```

Comment:
We should rename this too.

```suggestion
  const fileContent = fs.readFileSync(
```